### DISCUSSION TITLE
遇到错误，无法使用

### DISCUSSION THREAD

**Discussion by callacat** [Harry-zklcdc/go-proxy-bingai#3] [Q&A]
我换了不同的机器，网络环境部署go-proxy-bingai，但是一提问就是Sorry, looks like your network settings are preventing access to this feature.几天了，我人都麻了，有没有遇到同样问题解决了的，救救孩子吧

**Comment by yifangyujason** [comment]
手机端可以正常访问吗？可以的话请看https://github.com/adams549659584/go-proxy-bingai/issues/362

**Reply by callacat** [reply]
谢谢，手机也不可以访问

**Comment by Harry-zklcdc** [comment]
请在一个回复中贴出详细操作过程，并且说明机器所在地区

**Reply by callacat** [reply]
不想搞了，反正也就我自己用，直接用官网算了。我判断还是因为使用的白嫖的服务器，甲骨文、replit，这些应该都被bing ban了，我又不想专门为了这个去买干净IP的机器，不试了。谢谢
